cmd/allocator/main.go

```diff
@@ -135,6 +174,14 @@ func main() {
 	logger.WithError(err).Fatal("allocation service crashed")
 }
 
+func loadCACertPool() *x509.CertPool {
+	caCertPool, err := getCACertPool(certDir)
+	if err != nil {
+		logger.WithError(err).Fatal("could not get CA certs")
+	}
+	return caCertPool
```

Comment:
If there is an error will this just return nil?

Would it be better to leave caCertPool unmodified if there is an error parsing changes to the cert directory? Or have the process crash?

Right now this error gets logged and then has a potentially weird side effect.

Reply:
logger.WithError(err).**Fatal**(..) takes care of crashing.

Reply:
Yup, sorry I missed that.